scipy/spatial/transform/_rotation.pyx

```diff
@@ -910,8 +915,12 @@ cdef class Rotation:
         ----------
         quat : array_like, shape (N, 4) or (4,)
             Each row is a (possibly non-unit norm) quaternion representing an
-            active rotation, in scalar-last (x, y, z, w) format. Each
-            quaternion will be normalized to unit norm.
+            active rotation. The assumed component order depends on `scalar_first`
```

Comment:
I think this comment (and elsewhere) should be more explicit about which of the orders corresponds to `scalar_first = False`

Reply:
For it we have to give some meaningful definition for the components or give an example for an identity rotation. 

I think the latter approach will be very appropriate and easy to understand. I'll work on that.

Reply:
I ended up writing definitions from angle and axis of a rotation. See if you like it.

I have to say, that the whole docstring of `Rotation` needs a revision. It's not very informative or cleanly written.